homeassistant/components/waqi/sensor.py

```diff
@@ -141,8 +142,9 @@ def name(self):
     @property
     def native_value(self):
         """Return the state of the device."""
-        if self._data is not None:
-            return self._data.get("aqi")
+        if value := self._data.get("aqi"):
```

Comment:
Is 0 a valid value? The logic was changed for that case.

Reply:
https://github.com/home-assistant/core/pull/97767